server/utils/s3.js

```diff
@@ -112,3 +115,49 @@ export const uploadToS3FromUrl = async (url: string, key: string) => {
     }
   }
 };
+
+export const setS3ObjectAcl = () =>
+  ENABLE_PRIVATE_CONTENT ? 'private' : 'public-read';
+
+export const getUrlForImageProxy = (key: string) =>
+  `/api/images.info?key=${key}`;
+
+export const getSignedImageUrl = async (key: string) => {
+  const s3 = new AWS.S3({
```

Comment:
Seems like we could probably move the new `s3` object creation and the existing one https://github.com/outline/outline/pull/1137/files#diff-6445a4e1aa245e5905cb555afdd8ac6bR82-R87 out to the module context?

Reply:
yep, I'll give that a try. I'm bringing baggage from Python where the thread-safety of boto3 is inconclusive!

Reply:
Yea, I think the existing example in the code didn't do any favors either – I'm 90% sure it should be fine defined in the module context.